oak_containers_stage1/src/dice.rs

```diff
@@ -109,7 +106,7 @@ fn read_stage0_dice_data(start: PhysAddr) -> anyhow::Result<Stage0DiceData> {
             length.try_into()?,
             ProtFlags::PROT_READ | ProtFlags::PROT_WRITE,
             MapFlags::MAP_SHARED,
-            dice_file.as_raw_fd(),
+            Some(dice_file),
```

Comment:
I think this should be something like `Some(dice_file.as_fd())` or `Some(&dice_file)`. The current implementation consumes `dice_file`, which I think means that it is closed by the time we try to access it by reading from the `mmap`ed memory range.

Reply:
Thanks Conrad, great catch, that seems to be exactly the problem! :100: